engine/script/src/script_table.cpp

```diff
@@ -897,10 +886,14 @@ namespace dmScript
                     intptr_t aligned_buffer = ((intptr_t) offset + sizeof(float)-1) & ~(sizeof(float)-1);
                     intptr_t align_size = aligned_buffer - (intptr_t) offset;
                     buffer += align_size;
-                    // Sanity-check. At least 4 bytes alignment (de facto)
-                    assert((((intptr_t) buffer) & 3) == 0);
 
-                    lua_pushnumber(L, *((lua_Number_4_align *) buffer));
+                    union
+                    {
+                        lua_Number x;
+                        char x_buf[sizeof(lua_Number)];
+                    };
+                    memcpy(x_buf, buffer, sizeof(lua_Number));
+                    lua_pushnumber(L, x);
```

Comment:
UbSan reported this as a runtime error, since the pointer was 4 byte aligned, but the data type required an 8 byte alignment.